client/web/compose/src/views/Public/Pages/Records/View.vue

```diff
@@ -245,6 +265,29 @@ export default {
         this.$router.push({ name: 'page.record.edit', params: this.$route.params })
       }
     },
+
+    handleRedirectToPrevOrNext (value) {
+      const recordID = this.getNextOrPrevRecordId(this.recordID, value)
```

Comment:
Since you'll emit value you won't need to recompute it here, call it recordID

`if (!recordID) return`